app/src/main/java/eu/kanade/presentation/browse/ExtensionsScreen.kt

```diff
@@ -278,16 +281,47 @@ private fun ExtensionItem(
     onClickItemAction: (Extension) -> Unit,
     onClickItemSecondaryAction: (Extension) -> Unit,
     modifier: Modifier = Modifier,
+    context: Context,
 ) {
     val (extension, installStep) = item
+    var showUninstallConfirmation by remember { mutableStateOf(false) }
+
+    if (showUninstallConfirmation) {
+        ExtensionUninstallConfirmation(
+            extensionName = extension.name,
+            onClickConfirm = {
+                onLongClickItem(extension)
+                showUninstallConfirmation = false
+            },
+            onClickDismiss = {
+                showUninstallConfirmation = false
+            },
+            onDismissRequest = {
+                showUninstallConfirmation = false
+            },
+        )
+    }
+
     BaseBrowseItem(
         modifier = modifier
             .combinedClickable(
                 onClick = { onClickItem(extension) },
-                onLongClick = { onLongClickItem(extension) },
+                onLongClick = {
+                    if (context.isPackageInstalled(extension.pkgName)) {
+                        onLongClickItem(extension)
+                    } else {
+                        showUninstallConfirmation = true
+                    }
```

Comment:
```suggestion
                    if (extension is Extension.Installed && !extension.isShared) {
                        showUninstallConfirmation = true
                    } else {
                        onLongClickItem(extension)
                    }
```
otherwise it'll interfere with extension that were actually not installed